.travis.yml

```diff
@@ -0,0 +1,48 @@
+dist: bionic
+language: python
+jdk: openjdk11
+cache:
+  - pip
+  - directories:
+    - $HOME/.m2
+env:
+  global:
+  - AWS_REGION="us-east-1"
+  - AWS_DEFAULT_REGION=$AWS_REGION
+
+before_install:
+  - >
+    pip install --upgrade pip 'attrs==19.2.0'
+    -r https://raw.githubusercontent.com/aws-cloudformation/cloudformation-cli/master/requirements.txt
+install:
+  - pip install .
+
+script:
+  - pre-commit run --all-files
+
+jobs:
+  include:
+    - python: "3.6"
+      language: python
+    - python: "3.7"
+      language: python
+    - python: "3.8"
+      language: python
+    - language: java
+      before_install: skip
+      install: skip
+      script: mvn verify
+    - stage: integ
+      install:
+        - mvn install
+        - pip install .
+      script:
+        - DIR=$(mktemp -d)
+        - cd "$DIR"
+        - ls -la
+        - printf "AWS::Foo::Bar\n\n" | cfn init -vv
+        - mvn verify
+        - ls -la
+    - stage: deploy-maven
+      if: branch = master
+      script: skip
```

Comment:
can we remove this for now? it still takes 1+ minutes to spin up the worker, even with the skip condition

Reply:
yep was just trying some stuff out wasn't final